templates/vhost/_directories.erb

```diff
@@ -220,6 +220,9 @@
     <%- if directory['ssl_options'] -%>
     SSLOptions <%= Array(directory['ssl_options']).join(' ') %>
     <%- end -%>
+    <%- if directory['ssl_verify_client'] -%>
+    SSLVerifyClient <%= directory['ssl_verify_client'] %>
```

Comment:
Could you document this new value in the `apache::vhost::directories` docs? https://github.com/puppetlabs/puppetlabs-apache/blob/3.1.0/README.md#parameter-directories-for-apachevhost

Reply:
Will do!

Reply:
Hi @rmacd . Just following up on this - did you get a chance to add the new value in the docs? I can't see it. Thanks!